portality/formcontext/forms.py

```diff
@@ -370,6 +370,11 @@ class Workflow(Form):
                     ' completed your review.'
     )
 
+class Seal(Form):
+    """ Field to set the DOAJ Seal """
+
+    doaj_seal = BooleanField('Set DOAJ Seal', [validators.Optional()], description='description for doaj seal field.', false_values=(BooleanField.false_values + (False,)))
```

Comment:
"description for doaj seal field." ? I don't recall what "description" is used for, wasn't it the little gray text below fields though?